packages/compiler/src/testing/tester.ts

```diff
@@ -80,6 +81,10 @@ async function createTesterFs(base: string, options: TesterOptions) {
     if (resolved.type === "module") {
       const virtualPath = computeRelativePath(lib, resolved.mainFile);
       fs.addJsFile(virtualPath, host.getJsImport(resolved.mainFile));
+      fs.add(
+        resolvePath("node_modules", lib, "package.json"),
+        (resolved.manifest as any).file.text,
+      );
```

Comment:
If the library has no typespec entrypoint then it would fail to resolve things in `importPath` above and not register it. 

So here we might double add the package.json file (just will override the next time) but this make it work.

A cleaner solution woul maybe be to overall the library loading to get a bigger picture of what the library provide and then decide if you should load the sources.